aip_urdf_compiler/readme.md

```diff
@@ -0,0 +1,121 @@
+# aip_urdf_compiler
+
+## Overview
+
+The aip_urdf_compiler package provides tools for dynamically generating URDF (Unified Robot Description Format) files from configuration files during the build process. It simplifies sensor model management by automatically URDF models from sensor configurations.
+
+## Key Features
+
+- Dynamic URDF generation during colcon build
+- Automated sensor transform processing
+- Support for multiple sensor types and configurations
+
+## Usage
+
+### Package Integration
+
+To use aip_urdf_compiler in your description package:
+
+- Add the dependency in `package.xml`:
+
+  ```xml
+  <depend>aip_urdf_compiler</depend>
+  ```
+
+- Add the following to `CMakeLists.txt`:
+
+  ```cmake
+  find_package(aip_urdf_compiler REQUIRED)
+  aip_cmake_urdf_compile()
+  ```
+
+- Configure your sensors in `config/sensors.yaml` with required meta values:
+
+  - `type`: Sensor type identifier
+  - `frame_id`: TF frame identifier
```

Comment:
Could you refer `type` is required while `frame_id` is optional?
Also, those changes are required only in aip_launcher and not in individual_params.